app/experimenter/nimbus-ui/src/components/AppErrorBoundary/index.tsx

```diff
@@ -26,17 +26,21 @@ class AppErrorBoundary extends React.Component {
 
   render() {
     const { error } = this.state;
-    return error ? <AppErrorAlert /> : this.props.children;
+    return error ? <AppErrorAlert {...{ error }} /> : this.props.children;
   }
 }
 
-export const AppErrorAlert = () => {
-  return (
-    <Alert variant="warning" data-testid="error-loading-app">
-      <Alert.Heading>General application error</Alert.Heading>
-      <p>Something went wrong. Please try again later.</p>
-    </Alert>
-  );
-};
+export const AppErrorAlert = ({ error }: { error: Error }) => (
+  <Alert variant="danger" data-testid="error-loading-app">
+    <Alert.Heading>General application error</Alert.Heading>
+    <p>Something went wrong. Please try again later.</p>
+    {error?.message && (
```

Comment:
Seems like we want to keep any logic in here _super_ minimal since anything that errors in the furthest most outer error boundary will be unhandled, but this seemed safe enough to me.